src/main/java/com/shep/fourthlecture/models/Ticket.java

```diff
@@ -0,0 +1,193 @@
+package com.shep.fourthlecture.models;
+
+import com.shep.fourthlecture.annotations.NotNull;
+import com.shep.fourthlecture.annotations.NotNullWarningProcessor;
+import com.shep.fourthlecture.models.impl.Identifiable;
+import com.shep.fourthlecture.models.impl.Printable;
+import com.shep.fourthlecture.sharing.impl.Shareable;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+
+@Getter
+@Setter
+public class Ticket implements Identifiable, Printable {
+    @NotNull
+    private int id;
+    private String concertHall;
+    private String eventCode;
+    private Long time;
+    private boolean isPromo;
+    private char stadiumSector;
+    private double maxBackpackWeight;
+    private double price;
```

Comment:
type is not correct

Reply:
I'd like to clarify. The type is not correct in price field, right?